### PR TITLE
Trim trailing whitespace off of inputs and outputs [SATURN-1037]

### PR DESCRIPTION
**Pull request by petesantos**
Trimming whitespace is handled on save

### CODE REVIEW

**Review by zarsky-broad — COMMENTED** [review]
Two things:
1. When I talked to batch about this they recommended trimming whitespace off of both ends.
2. Would it be better if this happened on save, just mapping through all of them?

**Comment by panentheos** [comment]
I agree that doing this on save would probably be better, since there's also an edge case with the debounced input if you happen to type and tab away quickly.

File: src/pages/workspaces/workspace/workflows/WorkflowView.js [hunk 1]
```diff
@@ -856,9 +855,14 @@ const WorkflowView = _.flow(
     this.setState({ saving: true })
 
     try {
+      const trimInputOutput = _.flow(
+        _.update('inputs', _.mapValues(v => v.trim())),
+        _.update('outputs', _.mapValues(v => v.trim()))
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/workflows/WorkflowView.js]: `_.mapValues(_.trim)` for both?

**Review by zarsky-broad — APPROVED** [review]
One minor comment.

**Review by panentheos — APPROVED** [review]
Looks good.

**Pull request merged**